### DISCUSSION TITLE
Multivalue delimiter (##) not functioning in v19.0.1

### DISCUSSION THREAD

**Discussion by KrytenBoxHead** [keycloak/keycloak#13764] [Archive]
In v17.0.1 I can create a user attribute with multiple values delimited by '##'. I can then create a client scope with a mapper for the user attribute with the multi-valued switch enabled. This produces a token with the attribute name/key whose value is an array with  a comma separated list of the ## delimited values e.g. "groups":["apple","pear"].
In v19.0.1, the ## delimiters appear to have no effect and the token includes "groups":["apple##pear"].

I have tried alternative delimiters in case the workaround ## delimiter has been replaced e.g. "," and " ".

**Comment by lexcao** [comment]
It is a known issue with the new admin console.
And it was already fixed.
https://github.com/keycloak/keycloak-ui/issues/3022
